### DISCUSSION TITLE
Add localization for Chinese languages

### DISCUSSION THREAD

**Discussion by k25961918** [dialogic-godot/dialogic#1899] [Ideas]
Can you add a Chinese translation option to the menu in dialogic editor? I can help you translate it.

**Comment by mechPenSketch** [comment]
Translation for the editor seems to be in progress, so far: https://github.com/coppolaemilio/dialogic/issues/1534

**Comment by Jowan-Spooner** [comment]
Oh sorry, didn't see this before. Translations for the editor (for 2.x) are not yet planned as we want to get 2.0 out first. I have not planned things out for beyond the initial release. For a feature like this we really need to measure interest first as it's a lot of work. 

So thank you for the offer, but this will have to stay a distant future for now.
